docs/supported_formats/trivista.rst

```diff
@@ -0,0 +1,18 @@
+.. _trivista-format:
+
+TriVista
+--------
+
+Reader for spectroscopy data saved using the VistaControl software
+for TriVista spectrometers from Teledyne Princeton Instruments.
+Currently, RosettaSciIO can only read the ``.tvf`` format from TriVista.
+However, this format supports spectral maps and contains all relevant metadata.
+
+If `LumiSpy <https://lumispy.org>`_ is installed, ``LumiSpectrum`` will be
```

Comment:
``Luminescence`` is the ``signal_type``, while ``LumiSpectrum`` the class name.
See https://docs.lumispy.org/en/latest/user_guide/introduction.html
(needs to be corrected in JobinYvon reader as well)